server_installation/topics/operating-mode/crossdc/assembly-setting-up-crossdc.adoc

```diff
@@ -0,0 +1,27 @@
+[id="assembly-setting-up-crossdc"]
+:context: xsite
+= Setting Up Cross DC with {jdgserver_name} {jdgserver_version_latest}
+
+[role="_abstract"]
+Use the following procedures for {jdgserver_name} {jdgserver_version_latest} to perform a basic setup of Cross-Datacenter replication.
+
+This example for {jdgserver_name} {jdgserver_version} involves two data centers, `site1` and `site2`. Each data center consists of 1 {jdgserver_name} server and 2 {project_name} servers. We will end up with 2 {jdgserver_name} servers and 4 {project_name} servers in total.
```

Comment:
@andymunro jdgserver_version here resolves to 9.4.19. It should be 11.0.3

Reply:
maybe change it to jdgserver_version_latest

Reply:
Okay, will be in my next commit.